PerfectWeddingWeb/src/main/java/com/infoshareacademy/controllers/ProviderRestController.java

```diff
@@ -27,9 +27,9 @@ public ProviderRestController(ServiceProviderService serviceProviderService, Ser
         return serviceProviderService.findByCity(city);
     }
 
-    @GetMapping(value = "/service-provider/search-by-type/{service}")
-    public List<ServiceProvider> returnProvidersFoundByServiceType(@PathVariable String service) {
-        return serviceProviderService.findTypeOfService(service);
+    @GetMapping(value = "/service-provider/search-by-type/{service}{city}")
```

Comment:
Parametr obok parametru bez separatora nie zadziała, w URL będzie to potraktowane jako jeden ciąg znaków ponieważ nie zapisujemy nawiasów wąsatych w URL. Należy rozdzielić to jakimś separatorem.